src/fw/services/common/bluetooth/bluetooth_ctl.c

```diff
@@ -89,7 +89,9 @@ static void prv_comm_start(void) {
   stop_mode_disable(InhibitorCommMode);
   // Heap allocated to reduce stack usage
   BTDriverConfig *config = kernel_zalloc_check(sizeof(BTDriverConfig));
+#ifndef BT_NO_REQUIRE_ROOT_KEYS_GEN
```

Comment:
I'd suggest we just remove this completely, either here or via a TODO+tracker issue.